index.html

```diff
@@ -24,257 +24,305 @@
-              <li>Angular</li>
-            </ul>
+      <section class="about">
+        <h2 class="title title--about">About me</h2>
 
-          </div>
-        </li>
+        <p class="description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
 
-        <li class="about__card">
-
-          <div class="about__card__imgbox">
-            <img src="images/icon-skills.svg" alt="skills-icon" class="about__card__img">
-          </div>
-
-          <div class="card__details">
-
-            <h3 class="card__text card__text--about">Skills</h3>
+        <div class="about__btn">
+          <a href="#" class="btn btn--green">get me resume</a>
+        </div>
 
-            <ul class="about__card__list">
-              <li>Github</li>
-              <li>Git</li>
-              <li>Terminal</li>
-              <li>Figma</li>
-              <li>Design</li>
-            </ul>
+        <ul class="card-container">
 
-          </div>
-        </li>
-      </ul>
-    </section>
+          <li class="about__card">
 
+            <div class="about__card__imgbox">
+              <img src="images/icon-languages.svg" alt="Languages-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Languages</h3>
+
+              <ul class="about__card__list">
+                <li>javaScript</li>
+                <li>Ruby</li>
+                <li>Python</li>
+                <li>html</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-frameworks.svg" alt="frameworks-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Frameworks</h3>
+
+              <ul class="about__card__list">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Angular</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-skills.svg" alt="skills-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Skills</h3>
+
+              <ul class="about__card__list">
+                <li>Github</li>
+                <li>Git</li>
+                <li>Terminal</li>
+                <li>Figma</li>
+                <li>Design</li>
+              </ul>
+
+            </div>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="contact">
+
```

Comment:
- [ ] (Optional) It would be nice to use a more semantic tag like `section` instead of `footer` for this element. Ideally, the `footer` of a project contains **_copyright, rights reserved, logo, and social media icons/links_**.